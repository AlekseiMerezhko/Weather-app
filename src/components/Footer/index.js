import React from "react";

const Footer = () => {
  return <footer className="p-8 bg-teal-500"></footer>;
};

export default Footer;
