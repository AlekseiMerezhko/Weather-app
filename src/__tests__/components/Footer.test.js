import React from "react";
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";
import Footer from "../../components/Footer";

describe("<Footer />", () => {
  it("Check Footer render", () => {
    const footer = renderer
      .create(<Footer cities={{ loading: false }} logined={true} />)
      .toJSON();
    expect(footer).toMatchSnapshot();
  });
});
