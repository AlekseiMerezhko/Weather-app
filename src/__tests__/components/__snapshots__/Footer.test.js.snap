// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Check Footer render 1`] = `
<footer
  className="pt-4 pb-0 bg-teal-500"
>
  <div
    className="w-full md:flex justify-center text-white pl-4 pr-4 pb-2"
  >
    <div
      className="md:w-1/3 flex justify-center"
    >
      <div
        className="text-center"
      >
        <h2
          className="text-lg"
        >
          Application info
        </h2>
        <p>
          This is test app with react, redux, formik, yup and accuweather api
        </p>
      </div>
    </div>
    <div
      className="md:w-2/3 flex justify-around"
    >
      <div
        className="md:w-1/2 sm:w-1/2 flex justify-center"
      >
        <div>
          <h2
            className="text-lg"
          >
            SOCIAL
          </h2>
          <ul>
            <li
              className="hover:text-black cursor-pointer"
            >
              Facebook
            </li>
            <li
              className="hover:text-black cursor-pointer"
            >
              Linkedin
            </li>
            <li
              className="hover:text-black cursor-pointer"
            >
              Twitter
            </li>
            <li
              className="hover:text-black cursor-pointer"
            >
              Telegram
            </li>
            <li
              className="hover:text-black cursor-pointer"
            >
              YouTube
            </li>
          </ul>
        </div>
      </div>
      <div
        className="md:w-1/2 sm:w-1/2 flex justify-center"
      >
         
        <div>
          <h2
            className="text-lg"
          >
            WORK
          </h2>
          <ul>
            <li
              className="hover:text-black cursor-pointer"
            >
              Trello
            </li>
            <li
              className="hover:text-black cursor-pointer"
            >
              Slack
            </li>
            <li
              className="hover:text-black cursor-pointer"
            >
              GitHub
            </li>
            <li
              className="hover:text-black cursor-pointer"
            >
              Bitbucket
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex justify-center bg-teal-400 p-2"
  >
    <p>
      Â© 2020 Copyright: Weather-App
    </p>
  </div>
</footer>
`;
